doc/openms/docs/about/installation/installation-on-gnu-linux.md

```
@@ -70,6 +70,35 @@ If you encounter errors with unavailable packages, troubleshoot using the follow
 :start-after: "% start-after"
 ```
 
+## Reading Thermo Fisher RAW files
```

markdownlint-cli2 (0.23.0)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)